presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -245,6 +258,25 @@ static std::string maybeSetupTaskSpillDirectory(
   return spillPath;
 }
 
+void TaskManager::processResultRequests(
```

Comment:
You could call this method getData(...) also as processResultRequests makes it more vague.